app.go

```diff
@@ -126,6 +126,7 @@ func serve(port int, hc *resources.MapperHealthcheck, cth *resources.ContentTran
 		hc.ProducerQueueCheck(),
 	)
 	r.Handle("/content-transform/{uuid}", cth).Methods("POST")
```

Comment:
`/content-transform/{uuid}` is meant to be used by PAM. So feel free to remove this endpoint. 
I remember a discussion to use `/map` as endpoint instead. If we agreed to change endpoint name, I suggest to refactor  `ContentTransformHandler` in `MapEndpointHandler`.